src/styles/easy-clock.scss

```
@@ -0,0 +1,40 @@
+.easy-clock {
+  position: relative;
+  box-sizing: border-box;
+  width: #{$side};
+  height: #{$side};
+  background-color: #{$background-primary};
+  border: 0.5px solid black;
+  border-radius: 50%;
+
+  &__hour {
+    @extend %hands;
+    width: 50px;
+    transform: rotate(1deg);
+    animation: move-clock #{$duration-hour} linear infinite;
+  }
+  &__minute {
+    @extend %hands;
+    width: 80px;
+    animation: move-clock #{$duration-minute} linear infinite;
+  }
+  &__second {
+    @extend %hands;
+    width: 40px;
+    animation: move-clock #{$duration-second} linear infinite;
+  }
+  &__microsecond {
+    @extend %hands;
+    width: 20px;
+    animation: move-clock #{$duration-microsecond} linear infinite;
+  }
+}
+
+@keyframes move-clock {
+  from {
+    transform: rotate(-90deg);
+  }
+  to {
+    transform: rotate(269deg);
+  }
+}
```

move keyframes to the seperate file 
```suggestion
@keyframes hand-rotation {
  from {
    transform: rotate(-90deg);
  }
  to {
    transform: rotate(269deg);
  }
}
```
Why not from 0 to 360?

---

because it is usually horizontal block and hand on 0deg will be 3o'clock. animation should start from 12 o'clock so -90deg. -90 + 359 = 269deg

---

try `transofrm-origin: bottom`